.buildkite/test-pipeline.yaml

```diff
@@ -126,7 +126,9 @@ steps:
   - tests/distributed
   - tests/spec_decode/e2e/test_integration_dist_tp4
   - tests/compile
+  - examples/offline_inference/rlhf.py
   commands:
+  - python3 ../examples/offline_inference/rlhf.py
```

Comment:
Can we move this line to the bottom of this section and add a comment saying we should have a separate section for distributed examples in the future?

Reply:
fixed in [34cd83d](https://github.com/vllm-project/vllm/pull/12084/commits/34cd83dc4e43bef9af30d635c8215e5570a8f30c)